### DISCUSSION TITLE
Table表头之间的分隔线可以去掉吗

### DISCUSSION THREAD

**Discussion by whale-jung** [ant-design/ant-design#44266] [Q&A]
<img width="977" alt="image" src="https://github.com/ant-design/ant-design/assets/80868414/10746b45-8abe-45a1-94de-4c57d3e851fc">

**Comment by Yuiai01** [comment]
目前应该不支持通过 props 的形式来去掉分割线，不过你可以尝试使用 CSS 来覆盖。

**Reply by whale-jung** [reply]
ok了，谢谢

**Reply by lxiiiixi** [reply]
请问通过哪个 class 覆盖掉，好像没有找到。
